src/integrationTest/java/uk/gov/hmcts/reform/timedevent/infrastructure/config/SwaggerPublisher.java

```diff
@@ -8,26 +8,17 @@
 import java.nio.file.Paths;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit.jupiter.web.SpringJUnitWebConfig;
-import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.timedevent.testutils.SpringBootIntegrationTest;
 
-@SpringJUnitWebConfig
-@SpringBootTest
-@AutoConfigureMockMvc
 @SuppressWarnings("all")
-class SwaggerPublisher {
-
-    @Autowired
-    private MockMvc mvc;
+class SwaggerPublisher extends SpringBootIntegrationTest {
```

Comment:
I think the swagger generation is the same, but there's a new way to publish it in the pipeline which Lee has implemented in the other apps.  You may have see it already.

Reply:
I will apply Lee's changes

Reply:
I applied changes, I turned off Swagger Publisher for now until the app will be running in AAT with correct Flux, Jenkins configs, etc. I faced some permission errors on publish action probably because of missing setttings